datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemHeaderView.kt

```diff
@@ -58,7 +43,12 @@ internal class QuestionnaireItemHeaderView(context: Context, attrs: AttributeSet
     prefix.updateTextAndVisibility(questionnaireItem.localizedPrefixSpanned)
     updateQuestionText(question, questionnaireItem)
     hint.updateTextAndVisibility(questionnaireItem.localizedInstructionsSpanned)
-    initHelpButton(this, questionnaireItem)
+    initHelpButton(
+      findViewById(R.id.helpButton),
+      findViewById(R.id.helpCardView),
+      findViewById(R.id.helpText),
+      questionnaireItem
```

Comment:
```suggestion
    initHelpButton(
      helpButton = findViewById(R.id.helpButton),
      helpCardView = findViewById(R.id.helpCardView),
      helpText = findViewById(R.id.helpText),
      questionnaireItem
```

and in the later call sites.

Reply:
done.